server/game/cards/06-WoE/SibylWaimare.js

```diff
@@ -0,0 +1,30 @@
+const Card = require('../../Card.js');
+
+class SibylWaimare extends Card {
+    // At the start of your opponent's turn, that player discards the top card
+    // of their deck and exhausts each creature of that card's house.
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onBeginRound: (_, context) =>
+                    context.player.opponent === this.game.activePlayer &&
+                    context.player.opponent.deck.length > 0
+            },
+            gameAction: ability.actions.discard((context) => ({
+                target: context.player.opponent.deck[0]
+            })),
+            then: {
+                gameAction: ability.actions.sequentialForEach((context) => ({
```

Comment:
Here should not be sequentialForEach. Exhaust must happen at the same time. Just use exhaust with target

Reply:
Thanks, didn't realize that could be multiple cards.  Fixed!